test/test_utils.py

```diff
@@ -722,7 +731,96 @@ def test_load_commit_from_forked_repo(self):
         with self.assertRaisesRegex(
                 ValueError,
                 'If it\'s a commit from a forked repo'):
-            model = torch.hub.load('pytorch/vision:4e2c216', 'resnet18', force_reload=True)
+            model = torch.hub.load('pytorch/vision:4e2c216', 'resnet18', force_reload=True, trust_repo=True)
+
+    def _cleanup(self):
```

Comment:
Maybe we should make this a `TearDown()` method instead? https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown